### ISSUE TITLE
Notes on ESR141

### ISSUE THREAD

**Issue by sagudev** [servo/mozjs#476]
Continuation of #462.

Before it we would want to solve intl/bidi story, preferably in upstream or else we will have a problem, see https://github.com/servo/mozjs/issues/462#issuecomment-2220698183

...

**Comment by jschwe** [comment]
@sagudev Have you been following this further? Time flies and ESR 141 is not that far away anymore.

**Comment by sagudev** [comment]
Not really, I submitted a patch for ld but it got stale. Only other actionable item is intl/bidi that still needs fixing (but we will survive without it too), all other stuff are just notes to myself. I usually deal with ESR bump at the end of summer (that is also end of existing ESR cycle).

On that note it looks that next ESR will be 140 not 141.

**Comment by sagudev** [comment]
oh and i think we can drop https://github.com/servo/mozjs/blob/main/mozjs-sys/etc/patches/0032-macos-sdk-min-14-2.patch now because GH added newer xcode to macos images

**Comment by redfire75369** [comment]
`old-configure` has been removed, so `makefile.cargo` needs to be updated.
https://bugzilla.mozilla.org/show_bug.cgi?id=1957023

**Comment by jschwe** [comment]
@sagudev Can this issue be closed, given that 140 was the ESR and we are already on that?
